docs/upgrade-notes.asciidoc

```diff
@@ -48,6 +48,42 @@ For Elastic Security solution release information, refer to {security-guide}/rel
 [float]
 === Breaking changes
 
+[discrete]
+[[breaking-207091]]
+.Removed legacy security rules bulk endpoints (9.0.0)
+[%collapsible]
+====
+*Details* +
+--
+* `POST /api/detection_engine/rules/_bulk_create` has been replaced by `POST /api/detection_engine/rules/_import`
+* `PUT /api/detection_engine/rules/_bulk_update` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `PATCH /api/detection_engine/rules/_bulk_update has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `DELETE /api/detection_engine/rules/_bulk_delete` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+* `POST api/detection_engine/rules/_bulk_delete` has been replaced by `POST /api/detection_engine/rules/_bulk_action`
+--
+These changes were introduced in PR {kibana-pull}#197422[#197422]
+
+*Impact* +
+Deprecated endpoints will fail with a 404 status code starting from version 9.0.0
+
+*Action* +
+--
+Update your implementations to use the new endpoints:
+
+1. **For bulk creation of rules:**
+   - Use `POST /api/detection_engine/rules/_import` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-importrules[API documentation]) to create multiple rules along with their associated entities (e.g., exceptions and action connectors).
```

Comment:
@natasha-moore-elastic I need a sanity check here (please and thank you). The link in the preview improperly resolves to https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-update-fleet-proxies, instead of the URL above. Do you know why this is happening? Here's what I'm seeing:

![operation-update-fleet-proxies-url](https://github.com/user-attachments/assets/bb583663-5cb2-4d09-9dd3-b6f774f31a6d)

Reply:
```suggestion
   - Use `POST /api/detection_engine/rules/_import` (link:https://www.elastic.co/docs/api/doc/kibana/v8/operation/operation-importrules[API documentation]) to create multiple rules along with their associated entities (for example, exceptions and action connectors).
```

Reply:
I checked in incognito mode, and the link resolves properly (whew!). You're good, @nikitaindik! 

Also, 1+ to @natasha-moore-elastic's suggestion about using the `{api-kibana}` attribute instead of hard-coding the site's URL.